packages/peer-exchange/src/waku_peer_exchange_discovery.ts

```diff
@@ -15,7 +15,7 @@ import { PeerExchangeCodec, WakuPeerExchange } from "./waku_peer_exchange.js";
 const log = debug("waku:peer-exchange-discovery");
 
 const DEFAULT_PEER_EXCHANGE_REQUEST_NODES = 10;
-const PEER_EXCHANGE_QUERY_INTERVAL = 5 * 60 * 1000;
+const PEER_EXCHANGE_QUERY_INTERVAL = 10 * 1000;
```

Comment:
```suggestion
const PEER_EXCHANGE_QUERY_INTERVAL_MS = 10 * 1000;
```

So it's clear what unit we are looking at